platform/darwin/src/MGLStyleLayerManager.h

```diff
@@ -20,13 +20,27 @@ class LayerManagerDarwin : public LayerManager {
 
 private:
     LayerManagerDarwin();
+    /**
+     * @brief Enables a layer type for both JSON style and runtime API.
```

Comment:
You don't need `@brief` here - we use [Jazzy](https://github.com/realm/jazzy) to generate our documentation, and the only supported keywords (for Obj-C) are (from that doc): `@param`, `@return`, `@warning`, `@see` and `@note`

Reply:
By the way, we only use jazzy to generate developer-facing documentation about our public Objective-C API, not about internal C++ classes. The only consumer of this documentation comment would be Xcode’s built-in Quick Help feature, which supports either [HeaderDoc](https://developer.apple.com/library/archive/documentation/DeveloperTools/Conceptual/HeaderDoc/tags/tags.html#//apple_ref/doc/uid/TP40001215-CH346-SW14) or [a subset of Doxygen](https://stackoverflow.com/a/19169271/4585461). It doesn’t require `@brief` either.